servicetalk-opentracing-zipkin-publisher/src/main/java/io/servicetalk/opentracing/zipkin/publisher/ZipkinPublisher.java

```diff
@@ -278,6 +179,35 @@ public void onSpanFinished(final InMemorySpan span, long durationMicros) {
             builder.kind(Span.Kind.CLIENT);
         }
 
-        channel.writeAndFlush(builder.build());
+        Span s = builder.build();
+        reporter.report(s);
+    }
+
+    /**
+     * Blocking close method delegates to {@link #closeAsync()}.
+     */
+    @Override
+    public void close() {
+        closeable.onClose();
+    }
```

Comment:
`onClose()` isn't appropriate here as it does not actually close, also we need to subscribe here as `close()` is synchronous.
```suggestion
    public void close() throws Exception {
        closeable.closeAsync().toFuture().get();
    }
```

Reply:
@colestanfield I will just commit this suggestion

Reply:
Yikes ... compilation failure, Lemme fix this.